eng/common/pipelines/templates/steps/detect-api-changes.yml

```diff
@@ -6,7 +6,7 @@ parameters:
 
 steps:
   - pwsh: |
-      $apiChangeDetectRequestUrl = "https://apiview.dev/PullRequest/DetectApiChanges"
+      $apiChangeDetectRequestUrl = "https://apiview.dev/api/PullRequests/CreateAPIRevisionIfAPIHasChanges"
```

Comment:
This cannot go in until the service is updated and deployed.

Reply:
Do we plan to retire the older API?

Reply:
yes, I will coordinate this so that this is merged immediately after I deploy from this branch.
Yes the old API will be retired.